### PR TITLE
NIAD-333: vulnerable modules

### PR DESCRIPTION
**Pull request by Alicja-Kainos**
modify gradle dependencies in such way, that there are no vulnerable modules reported bu snyk monitor tool.

Comments and screens in JIRA ticket https://gpitbjss.atlassian.net/browse/NIAD-333 .

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,14 @@ configurations {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
-	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation( 'org.springframework.boot:spring-boot-starter-web:2.3.2.RELEASE') {
```
Comment (grzegorz-robak) [inline build.gradle]: shouldn't you update the gradle plugin: https://plugins.gradle.org/plugin/org.springframework.boot

**Review by grzegorz-robak — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (Alicja-Kainos) [reply build.gradle]: you're right, now the right version of tomcat is loaded implicitly

**Review by jaroslaw-wojciechowski — DISMISSED** [review]

File: build.gradle [hunk 2]
```diff
@@ -46,7 +46,7 @@ configurations {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
-	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-web:2.3.2.RELEASE'
```
Comment (mattd-kainos) [inline build.gradle]: shouldn't need this now that you've bumped the plugin version

File: responder.gradle [hunk 3]
```diff
@@ -39,7 +39,7 @@ configurations {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
-	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-web:2.3.2.RELEASE'
```
Comment (mattd-kainos) [inline responder.gradle]: same here, inherits versions from the plugin

**Review by grzegorz-robak — APPROVED** [review]

**Review by jaroslaw-wojciechowski — APPROVED** [review]

**Pull request merged**